index.html

```diff
@@ -0,0 +1,320 @@
+                <li><a href="index.html#10" class="nav-link">if...else statement</a></li>
+                <li><a href="index.html#11" class="nav-link">While statement</a></li>
+                <li><a href="index.html#12" class="nav-link">Function declarations</a></li>
+                <li><a href="index.html#13" class="nav-link">Reference</a></li>
+            </ol>     
+        </nav>
+    </div>
+    <div>
+    <main id="main-doc">
+        <section id="Introduction" >
+            <h1 name ="Introductio">Introduction</h1>
+            <p>JavaScript is a cross-platform, object-oriented scripting language. It is a small and lightweight language. Inside a
+                host environment (for example, a web browser), JavaScript can be connected to the objects of its environment to
+                provide
+                programmatic control over them.</p>
+            <br>
+            
+            <p>JavaScript contains a standard library of objects, such as Array, Date, and Math, and a core set of language elements
+                such as operators, control structures, and statements. Core JavaScript can be extended for a variety of purposes by
+                supplementing it with additional objects; for example:</p>
+            
+            <ul>
+                <li>
+                    <p>Client-side JavaScript extends the core language by supplying objects to control a browser and its Document
+                        Object Model
+                        (DOM). For example, client-side extensions allow an application to place elements on an HTML form and
+                        respond to user
+                        events such as mouse clicks, form input, and page navigation.</p>
+                </li>
+                <br>
+                <li>
+                    <p>Server-side JavaScript extends the core language by supplying objects relevant to running JavaScript on a
+                        server. For
+                        example, server-side extensions allow an application to communicate with a database, provide continuity of
+                        information
+                        from one invocation to another of the application, or perform file manipulations on a server.</p>
+                </li>
+            </ul>
+        </section>
+       <section id="What_you_should_already_know"><h1 id="1">What you should already know</h1> 
+    <p>This guide assumes you have the following basic background:</p>
+    <ul>
+        <li>A general understanding of the Internet and the World Wide Web (WWW).</li>
+        <li>Good working knowledge of HyperText Markup Language (HTML).</li>
+        <li>Some programming experience. If you are new to programming, try one of the tutorials linked on the main page
+            about
+            JavaScript.</li>
+    </ul>
+</section>
+<section id="JavaScript_and_Java">
+    <h1 id="2">JavaScript and Java</h1>
```

Comment:
Please make sure to only have on h1 per page as it makes it easier for the search engines to determine the main header of your page. For any other header on your page, you can h2-h6